### DISCUSSION TITLE
SNES Retroscaler issues Mod Power

### DISCUSSION THREAD

**Discussion by fmunozs** [darthcloud/BlueRetro#970] [Q&A]
I got a SNES Retroscaler BlueRetro adapter, it seems to work OKayis, I suspect its draining a lot of power from the SNES since sometimes the system reboots or locks up, mainly when I use it at the same time with the sd2snes (known to draw more power than an standard cartridge). I am attaching some pictures of the board. I was able to update it to 1.9.2 but the problems still occur.

What would you recommend me? I will try swapping SNES PSU and recapping the SNES it ASAP but I was wondering if its possible to mod the BlueRetro and use an external power supply for the ESP32, would not mind doing some solder work on it.

![image](https://github.com/darthcloud/BlueRetro/assets/1041604/c3befdde-ccfa-475c-baec-e535bf5aa8a4)
![image](https://github.com/darthcloud/BlueRetro/assets/1041604/ff43dfba-a3ab-48f1-b55c-d6ef98a228de)

**Comment by darthcloud** [comment]
I'm not familiar with RetroScaler design but you can probably cut the 5v coming from console and wire in a USB cable to power it up.

**Reply by fmunozs** [reply]
I got a new power supply for the snes and did not have to do any modding!
